go/vt/sqlparser/ast.go

```diff
@@ -1276,51 +1264,38 @@ func (node *ColName) Equal(c *ColName) bool {
 	return node.Name.Equal(c.Name) && node.Qualifier.Equal(c.Qualifier)
 }
 
+// ValTuple represents a list of value expressions.
+// It can be RowTuple or ParenExpr.
+type ValTuple interface {
+	iRowTuple()
+	Expr
+}
+
+func (RowTuple) iRowTuple()  {}
+func (ParenExpr) iRowTuple() {}
```

Comment:
Take a look at this diff: https://github.com/sougou/vitess/commit/11c55ddddce8c89b973ee8a3e3f1a0c4d5ac45f3. This one compiles for me.